docs-2.0/3.ngql-guide/10.tag-statements/2.drop-tag.md

```diff
@@ -4,9 +4,9 @@
 
 A vertex can have one or more tags.
 
-- If a vertex has only one tag, the vertex **CANNOT** be accessed after you drop it. But its edges are available. The vertex will be deleted in the next compaction.
+- If a vertex has only one tag, the vertex **CANNOT** be accessed after you delete it. But its edges are available. The vertex will be deleted in the next compaction.
 
-- If a vertex has multiple tags, the vertex is still accessible after you drop one of them. But all the properties defined by this dropped tag **CANNOT** be accessed.
+- If a vertex has multiple tags, the vertex is still accessible after you delete one of them. But all the properties defined by this deleted tag **CANNOT** be accessed.
```

Comment:
In the nGQL syntax, deleteing a tag means detaching a tag from a vertex, and dropping a tag means removing a tag from the graph space. Looks like it should be "drop" here?